templates/taxi/manufacturer/manufacturer_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2>Delete manufacturer</h2>
+  <form method="post">
+    {% csrf_token %}
+    <p>Are you sure you want to delete "{{ object }}"?</p>
+    {{ form }}
+    <input class="btn btn-primary" type="submit" value="Confirm">
```

Comment:
Add a cancel button beside the Confirm as mentioned [in the check list](https://github.com/AlexPelevin99/py-taxi-service-forms/blob/dev/checklist.md#code-style). It should return the user to the previous page. This button should be present in the car deletion as well.